src/components/board.js

```diff
@@ -1,22 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-export const Board = props => {
-    return (
-        <div className="root">
-            <div className="grid-container">{props.lights}</div>
-            <p>{props.numMoves}</p>
-            <button onClick={props.reset}>Reset</button>
-            {props.hasWon && (
-                <div>Congratulations, you Won! Click reset to play again.</div>
-            )}
-        </div>
-    );
-};
+const Board = props => (
+    <div className="root">
+        <div className="grid-container">{props.lights}</div>
+        <p>{props.numMoves}</p>
+        <button data-testid="reset" onClick={props.reset}>
+            Reset
+        </button>
+        {props.hasWon && (
+            <div data-testid="win-div">
+                Congratulations, you Won! Click reset to play again.
```

Comment:
Curious about @solomonhawk's thoughts on this — usually, if I have a long string like this and need to test for it, I end up setting/exporting the string as a const like `WIN_TEXT = 'Congratulations....'`. That way, there's no duplication and changing the text in `board.js` won't cause the test to break.